2022-02-27 23:46:55 ERROR (SyncWorker_0) [homeassistant.util.yaml.loader] expected '<document start>', but found '{'
  in "/config/templates/speech/example.yaml", line 2, column 1
2022-02-27 23:46:55 ERROR (MainThread) [homeassistant.bootstrap] Failed to parse configuration.yaml: expected '<document start>', but found '{'
  in "/config/templates/speech/example.yaml", line 2, column 1. Activating safe mode
2022-02-27 23:47:15 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.6.245
2022-02-27 23:47:19 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.10.9
2022-02-27 23:47:21 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.19.244
2022-02-27 23:47:29 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.19.244
2022-02-27 23:47:41 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.10.9
2022-02-27 23:47:51 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [22467924573392] Received invalid command: mobile_app/push_notification_channel
2022-02-27 23:48:19 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.6.245
2022-02-27 23:48:31 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.10.9
2022-02-27 23:48:34 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook 1dd4c955c8f7e53c836c4adc8d43c49136326c7e0452244b61add63efddf9c31 from 192.168.1.103
2022-02-27 23:48:38 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.10.9
2022-02-27 23:49:03 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.10.9
2022-02-27 23:49:06 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.6.245
2022-02-27 23:49:08 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.10.9
2022-02-27 23:49:22 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.6.245
2022-02-27 23:49:31 WARNING (SyncWorker_6) [homeassistant.util.yaml.loader] YAML file /config/themes/caule-themes-pack-1/mushroom.yaml contains duplicate key "mush-title-font-size". Check lines 9 and 10
2022-02-27 23:49:40 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook f1ca294f1e42d676a84c0a06c441a9f16559b553ccd319a3ad0b5aff5456296d from 52.14.19.244
2022-02-27 23:49:50 WARNING (SyncWorker_9) [homeassistant.util.yaml.loader] YAML file /config/themes/caule-themes-pack-1/mushroom.yaml contains duplicate key "mush-title-font-size". Check lines 9 and 10
2022-02-27 23:49:51 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 98, in forwarded_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 79, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 219, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 137, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/config/core.py", line 30, in post
    errors = await async_check_ha_config_file(request.app["hass"])
  File "/usr/src/homeassistant/homeassistant/config.py", line 928, in async_check_ha_config_file
    res = await check_config.async_check_ha_config_file(hass)
  File "/usr/src/homeassistant/homeassistant/helpers/check_config.py", line 214, in async_check_ha_config_file
    p_integration = await async_get_integration_with_requirements(
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 83, in async_get_integration_with_requirements
    await _async_process_integration(hass, integration, done)
  File "/usr/src/homeassistant/homeassistant/requirements.py", line 98, in _async_process_integration
    if integration.requirements:
AttributeError: 'Event' object has no attribute 'requirements'
2022-02-27 23:49:51 WARNING (MainThread) [homeassistant.components.template.sensor] The 'entity_id' option near /config/packages/canadian_weather.yaml:12 is deprecated, please remove it from your configuration
2022-02-27 23:49:51 WARNING (MainThread) [homeassistant.components.template.sensor] The 'entity_id' option near /config/packages/canadian_weather.yaml:31 is deprecated, please remove it from your configuration
2022-02-27 23:49:52 ERROR (MainThread) [homeassistant.components.homeassistant] The system cannot restart because the configuration is not valid: Invalid config for [alert]: [garage_after_dark] is an invalid option for [alert]. Check: alert->alert->garage_after_dark->garage_after_dark. (See /config/configuration.yaml, line 24). 
2022-02-27 23:49:52 ERROR (MainThread) [homeassistant.core] Error executing service: <ServiceCall homeassistant.restart (c:6390caa781b6b8ce94338309ab21d20e)>
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/core.py", line 1648, in catch_exceptions
    await coro_or_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1667, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 738, in admin_handler
    await result
  File "/usr/src/homeassistant/homeassistant/components/homeassistant/__init__.py", line 172, in async_handle_core_service
    raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: The system cannot restart because the configuration is not valid: Invalid config for [alert]: [garage_after_dark] is an invalid option for [alert]. Check: alert->alert->garage_after_dark->garage_after_dark. (See /config/configuration.yaml, line 24). 
